contrib/platform/src/com/sun/jna/platform/win32/WinNT.java

```diff
@@ -1988,13 +1988,13 @@ public byte getProductType() {
         }
     }
 
-    int VER_EQUAL = 1;
-    int VER_GREATER = 2;
-    int VER_GREATER_EQUAL = 3;
-    int VER_LESS = 4;
-    int VER_LESS_EQUAL = 5;
-    int VER_AND = 6;
-    int VER_OR = 7;
+    byte VER_EQUAL = 1;
+    byte VER_GREATER = 2;
+    byte VER_GREATER_EQUAL = 3;
+    byte VER_LESS = 4;
+    byte VER_LESS_EQUAL = 5;
+    byte VER_AND = 6;
+    byte VER_OR = 7;
```

Comment:
Is this a source/binary compatible change (will code compiled against 5.2 still work with 5.3 (or vice-versa)?  I see why you do it, but if if is not compatible, please stay with int.

Reply:
I searched the JNA codebase and don't see anywhere else where these constants are referenced.   

External code using JNA 5.2 or earlier may use these as `int`, but a `byte` will work anywhere an `int` is expected with a widening conversion transparent to the user.  The alternative is to require users to explicitly cast these all to `byte` to use them with `VerSetConditionMask` (and potentially other functions).

Given that byte-to-int happens transparently to the user, but int-to-byte requires an explicit cast, I do not think this breaks compatibility and `byte` is preferred here.

Reply:
Well apparently it's [not backwards compatible](https://stackoverflow.com/questions/54082926/can-i-change-constant-from-int-to-byte-in-java-without-breaking-backward-compati/54083710#54083710), which surprises me!  But I'll put it back to `int` and then cast.